projects/media-viewer/src/lib/annotations/comment-set/comment/comment.model.ts

```diff
@@ -5,4 +5,8 @@ export interface Comment extends ApiPersisted {
   annotationId: string;
   content: string;
   tags: TagItemModel[];
+  page: number;
+  pageHeight: number;
```

Comment:
its kept here but it was removed in the input for the comment component

Reply:
yeah that has moved now from component input to comment object